perf/scripts/parse_liboqs_speed.py

```diff
@@ -13,6 +14,11 @@ class State(Enum):
 data={}
 # fetch both x86 and aarch64 CPU details:
 data["cpuinfo"]=getcpuinfo(["flags", "model name", "cpu MHz", "Features", "CPU implementer", "CPU variant", "CPU part", "BogoMIPS"])
+estfrequency = getestimatedcpufrequency()
+if estfrequency != None:
+    data["cpuinfo"]["estfrequency"] = estfrequency
+else:
+    data["cpuinfo"]["estfrequency"] = "Unavailable"
```

Comment:
Well this isn't quite right, is it? `estfrequency` could also be obtained on x64 but the `if` statement above prohibits this.